compose/install/index.md

```diff
@@ -9,6 +9,10 @@ redirect_from:
 
 On this page you can find information on how to get and install Compose.
 
+> **Give Feedback!**
+>
+> [Give us feedback](#where-to-go-next) on this page and get entered into a lottery for Docker swag!
```

Comment:
This should point to the 'Feedback' section. It currently points to another section.

Also some minor suggestion about wording: Provide feedback about Docker Compose installation instructions and enter into a lottery for Docker swag!

Reply:
@usha-mandya 

This was not accidental,  if you use #feedback the section header will not be in view and it is confusing. 

**Using #feedback**
![image](https://user-images.githubusercontent.com/103652162/188925800-de16c0f3-d33b-49e4-953d-22e45322eb32.png)

If I use the previous one I see the whole "Feedback" section header and all, so this works best for users to situate themselves. 

**Using the previous section #**
![image](https://user-images.githubusercontent.com/103652162/188925963-a895017f-14d2-44cf-8a5e-e431b83241e2.png)

I think "this page" is shorter and is leaves it lighter (whereas comment on the instructions might seem a more involved process)and since the context is the page where the note is at the top of it does not require clarification. 
Also we use that wording in the section where the survey link is available from. Are you too set on this suggestion? 
If not, I would just adjust the note's case from title to sentence case.